docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -429,7 +926,8 @@ The decoded contents of the `token` will look something like this:
 
 <CodeExample>
 
-```json
```

Comment:
I don't think we should do this, since this isn't a transaction and is in a different section.

Reply:
I went overboard replacing json examples content 🤦🏼